default.xml

```diff
@@ -27,14 +27,29 @@
   <project path="packages/apps/LineageParts" name="android_packages_apps_LineageParts" remote="maruos" />
   <project path="prebuilts/busybox" name="platform_prebuilts_busybox" remote="maruos" revision="master" />
 
-
   <project path="vendor/maruos" name="vendor_maruos" remote="maruos" />
   <project path="vendor/maruos/mflinger" name="mflinger" remote="maruos" />
   <project path="vendor/maruos/blueprints" name="blueprints" remote="maruos" revision="master" />
   <project path="vendor/maruos/installer" name="installer" remote="maruos" revision="master" />
-
+  
   <!-- Devices -->
+  <project name="LineageOS/android_packages_resources_devicesettings" path="packages/resources/devicesettings" remote="github" revision="lineage-16.0" />
+
+  <!-- START hammerhead - Nexus 5 -->
   <project path="kernel/hammerhead" name="android_kernel_hammerhead" groups="hammerhead" remote="maruos" clone-depth="100"/>
   <project path="device/lge/hammerhead" name="android_device_lge_hammerhead" groups="device,hammerhead,broadcom_pdk,generic_fs" remote="maruos" />
+  <!-- END hammerhead - nexus 5 -->
+
+  <!-- START pme - HTC 10 -->
+  <project path="device/htc/pme" name="android_device_htc_pme" remote="maruos" />
+  <project path="kernel/htc/msm8996" name="android_kernel_htc_msm8996" remote="maruos" />
+  <project path="vendor/htc" name="proprietary_vendor_htc" remote="maruos" revision="lineage-16.0" />
+  <!-- END pme - HTC 10 -->
+
+    <!-- START Sailfish/Marlin - Google Pixel / Google Pixel XL -->
+  <project path="device/google/marlin" name="android_device_google_marlin" remote="maruos" />
+  <project path="kernel/google/marlin" name="android_kernel_google_marlin" remote="maruos" />
+  <project path="vendor/google" name="proprietary_vendor_google" remote="maruos" revision="lineage-16.0" />
```

Comment:
We don't usually include the vendor blob repos in our manifest and usually let devs select whichever method they want to use of obtaining these (including stripping them from factory images with either android-generate-vendor or android-prepare-vendor).

But having the vendor blobs forked in and set up by default in our manifest may make our community's development efforts easier since devices just build out of the box. I don't mind trying this approach for maru-0.7.

Any thoughts, @utzcoz?

Reply:
if I may, advanced devs, know how to get blobs from other sources if they need to.
Including them eases things for people just getting starting in building/porting as it was my case.

Maru clearly needs support and committed people. Making things as easy as possible is a way to attract / keep devs.

Reply:
Vendor proprietary blob maybe incur `DMCA` from GitHub. What about we update the porting document with the method to get and add those files to specific devices, and add document to our Q&A part to devs?

Reply:
If DMCA is a problem and I understand it can be, maybe we should not host the blobs in the maru org, but still include links to some proprietary repo so that devices can build out of the box. For example https://github.com/TheMuppets/
This was if some company has to complain against it's blobs being hosted they will ask github to clos TheMuppets repo.
So no risks, but clearly improved developer experience.

Reply:
I can see arguments both ways. Yes, the dev experience is easier since devices will build out of the box, but on the downside devs will not know the process for obtaining vendor blobs themselves, which is important knowledge if you want to know what's actually happening under the hood.

My main issue with including links to those specific vendor repos in our official manifest may be seen as endorsing them, which we do not do in any way. They are useful to get going quickly, but the files really should be pulled from a trusted factory image or running device.

Given that LineageOS itself does not ship these blob files in their official manifest, I think it's reasonable to assume that Maru OS devs should learn how to obtain these on their own. As @utzcoz suggests, we can update our developer docs with very simple and clear instructions on how to get these blobs, and in the device-specific guide (e.g. [Pixel](https://maruos.com/docs/devices/sailfish.html#install-maru-via-twrp)) we can give explicit instructions on how to obtain the blobs for that device. Also, for the Google devices like the Pixel, we usually use [android-prepare-vendor](https://github.com/anestisb/android-prepare-vendor) to grab these blobs from the official factory image, which is a trusted source.

Sorry @pintaf, but let's take the conservative approach for now. Let's remove the forked blob repos from our org, and remove the manifest links. I'm happy to merge everything else. We can discuss this further later too, but let's at least merge the rest of the PR while we figure out how best to deal with this.

Reply:
Ok @pdsouza I removed the vendor blobs. Ill try to delete the associated repos from maru org-
Would be nice if you can merge that I have some time to work on this tonight.